main/main.cpp

```diff
@@ -1977,6 +1977,37 @@ Error Main::setup2(Thread::ID p_main_tid_override) {
 			}
 		}
 	}
+
+	if (editor) {
+		bool restore_editor_window = GLOBAL_DEF("editor/run/remember_window_size_and_position", false);
```

Comment:
I think this behavior should be enabled by default. Most people expect this to happen by default, judging by the issues and proposals that were opened over the years.

Reply:
I agree, but I decided to err on the side of caution by leaving the default behaviour unchanged. However, if the general consensus is that keeping the window settings is the preferable default, I am very happy to change it